pkg/keyidentifier/testdata/make_temp.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+
+# FIXME: this entire file should be deleted and merged with make.sh
+set -e
+
+function rand {
+    dd if=/dev/random bs=1 count=16 2> /dev/null  | base64
+}
+
+function makeUUID {
```

Comment:
Erm. When I said "uuid" I meant "random string". Please don't shell to ruby. Just use rand as defined above. Or something even simpler, so filter out the / characters. Or change rand to do both. We don't need strong randomness for any of this

Reply:
ha I figured the shell out to ruby would make you wince, meant to take it out in favor of some commented out shell script similar to the `rand` function. 

I'll simplify this vastly though, good call.